migrations/4_renounceAdmin.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2020 ChainSafe Systems
+ * SPDX-License-Identifier: LGPL-3.0-only
+ */
+
+const Helpers = require('../test/helpers');
+const Utils = require('./utils');
+
+const AccessControlSegregatorContract = artifacts.require("AccessControlSegregator");
+const FeeRouterContract = artifacts.require("FeeHandlerRouter");
+const BasicFeeHandlerContract = artifacts.require("BasicFeeHandler");
+const FeeHandlerWithOracleContract = artifacts.require("FeeHandlerWithOracle");
+
+module.exports = async function(deployer, network) {
+    const networksConfig = Utils.getNetworksConfig()
+    let currentNetworkConfig = networksConfig[network.split("-")[0]];
+
+    if (!currentNetworkConfig.access) return;
+
+    // fetch deployed contracts addresses
+    const accessControlInstance = await AccessControlSegregatorContract.deployed();
+    const feeRouterInstance = await FeeRouterContract.deployed();
+    const basicFeeHandlerInstance = await BasicFeeHandlerContract.deployed();
+    const feeHandlerWithOracleInstance = await FeeHandlerWithOracleContract.deployed();
+
+    if (currentNetworkConfig.access.feeHandlerAdmin) {
+      console.log("Renouncing handler admin to %s", currentNetworkConfig.access.feeHandlerAdmin);
+
+      await basicFeeHandlerInstance.renounceAdmin(currentNetworkConfig.access.feeHandlerAdmin);
+      await feeHandlerWithOracleInstance.renounceAdmin(currentNetworkConfig.access.feeHandlerAdmin);
+    }
+
+    if (currentNetworkConfig.access.feeRouterAdmin) {
+      console.log("Renouncing router admin to %s", currentNetworkConfig.access.feeRouterAdmin);
+
+      await feeRouterInstance.grantRole("0x00", currentNetworkConfig.access.feeRouterAdmin);
+      await feeRouterInstance.renounceRole("0x00", await deployer['networks'][deployer['network']]['from']);
+    }
+
+    for (let i = 0; i < currentNetworkConfig.access.accessControl.functions.length; i++) {
+      const func = currentNetworkConfig.access.accessControl.functions[i]
```

Comment:
can we make config a map? 
it would be difficult to maintain as a list

Reply:
Had to do it like that on solidity so forgot it was an option :sweat_smile: